fullscreen.bs

```diff
@@ -155,8 +155,12 @@ steps:
 <h2 id=api>API</h2>
 
 <pre class=idl>
+dictionary <dfn>FullscreenOptions</dfn> {
+  boolean prefersNavigationBar = true;
```

Comment:
As a rule of thumb, to which there are only a few exceptions, optional boolean dictionary members should default to false. This is because undefined is a falsy value, but treated the same as missing, which means that `{ prefersNavigationBar: undefined }` behaves like `{ prefersNavigationBar: true }`, which is surprising.

Is there another possible name which would invert the default? An enum is also an option which gets around the problem.